scripts/load_testing/client_config.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+from math import ceil
+import os
+
+
+INSTANCES = 3
+PREFIX = 'loadgen'
+RPCPERF_THREADS = 1
+RPCPERF_CONNS = 100
+RPCPERF_RATE = 10000
+RPCPERF_GET_RATIO = 0.9
+RPCPERF_SET_RATIO = 1 - RPCPERF_GET_RATIO
+PELIKAN_SLAB_MEM = 4294967296
+PELIKAN_ITEM_OVERHEAD = 48
+KSIZE = 32
+VSIZE = 32
+PELIKAN_SERVER_PORT = 12300
+
+
+def generate_config(rate, connections, vsize, slab_mem, threads):
+# create rpcperf.toml
+  nkey = int(ceil(1.0 * slab_mem / (vsize + KSIZE + PELIKAN_ITEM_OVERHEAD)))
+  conn_per_thread = connections / threads
+
+  config_str = '''
+[general]
+clients = {threads}
+tcp-nodelay = true
+poolsize = {connections} # this specifies number of connection per thread
+# runtime ~= windows x duration
+windows = 2
+interval = 60
+request_ratelimit = {rate}
+
+[[keyspace]]
+length = 32
+count = {nkey}
+weight = 1
+commands = [
+    {{action = "get", weight = 9}},
```

Comment:
Weights should be calculated from the get/set ratio (or those parameters converted to weight instead of ratio)

Reply:
switched to explicit weight.